.eslintrc

```diff
@@ -0,0 +1,53 @@
+{
+  "ecmaFeatures": {
+    "jsx": true,
+    "modules": true
+  },
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true
+  },
+  "parser": "babel-eslint",
+  "rules": {
+    "eol-last": 1,
+    "no-unused-vars": [2, {"varsIgnorePattern": "^React$"}],
+    "max-len": [1, 120, 4],
```

Comment:
I tried to avoid any eslint rule debates by using exactly what was already defined in dashboard :P